### DISCUSSION TITLE
Convert VARCHAR to LIST

### DISCUSSION THREAD

**Discussion by ghost** [duckdb/duckdb#12623] [Q&A] [answered]
In PostgreSQL `SELECT '{1,2,3}'::integer[]` works as expected but in DuckDb it gives this error:
```
ConversionException: Conversion Error: Type VARCHAR with value '{1,2,3}' can't be cast to the destination type LIST
```
It seems you can do it as 
```
SELECT regexp_extract_all('{1,2,3}', '\d+')::integer[]
```
but is there a simpler way?

**Comment by Tishj** [comment, marked as answer]
`select '[1,2,3]'::INTEGER[]`

**Reply by ghost** [reply]
Because I don't get to choose the inputs. The strings are from a dump.
